src/core/server/graph/resolvers/User.ts

```diff
@@ -7,16 +7,14 @@ import { roleIsStaff } from "coral-server/models/user/helpers";
 
 import {
   GQLFEATURE_FLAG,
-  GQLUser,
   GQLUSER_ROLE,
   GQLUserTypeResolver,
 } from "coral-server/graph/schema/__generated__/types";
 
 import { RecentCommentHistoryInput } from "./RecentCommentHistory";
 import { UserStatusInput } from "./UserStatus";
-import { getRequestedFields } from "./util";
 
-const maybeLoadOnlyIgnoredUserID = (
+const maybeLoadOnlyIgnoredUserID = async (
```

Comment:
What do you think of us renaming this to `loadAndFilterForExistingUsers`?

I know it sounds a little absurd, but we are indeed "loading the users" and then "filtering out null and errored users".

And since we no longer return only the user ID's, it might be confusing for some folks that it's named one thing and does another thing.